recipe/meta.yaml

```diff
@@ -20,7 +19,7 @@ build:
   - vc14  # [win and py35]
 
 # https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/issues/29
-  skip: true  # [win and py35]
+  #skip: true  # [win and py35]
```

Comment:
Though that seems to be for OpenMesh 7
graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/commit/90a75f0fb09ca49eab6149c28388bd7fa56e7bc9